package.json

```diff
@@ -44,6 +44,7 @@
     "hoist-non-react-statics": "^3.3.2",
     "lodash.isequal": "^4.5.0",
     "opencollective-postinstall": "^2.0.3",
+    "react-native-elements": "^1.0.0-beta8",
```

Comment:
Don't change the the package.json or yarn.lock.

Reply:
i didn't change it deliberately ,it changed automatically after yarn install . yeah sure  i will undo it

Reply:
do `git add src/searchbar/SearchBar-default.tsx`